web/src/app/app.component.ts

```diff
@@ -46,10 +46,11 @@ export class AppComponent implements OnInit {
           this.createUserSocialDetails(socialDetails);
           return;
         }
-        this.updateUserSocialDetails(response.id, socialDetails);
+        this.updateUserSocialDetails(response.id, { ...socialDetails, ...response });
       });
   }
 
+
```

Comment:
This extra line is not needed, but we can sort out in the next PR

Reply:
sure 👍